src/spead.py

```diff
@@ -213,6 +213,8 @@ def __init__(self, from_string=None, id=None, name='', description='', shape=[],
                     self.dtype_str = self._dtype_pack(ndarray)
                     self.shape = ndarray.shape if isinstance(ndarray, numpy.ndarray) else ndarray[1]
                     self.size = numpy.multiply.reduce(self.shape)
+                    if isinstance(ndarray, numpy.ndarray):
+                        self.fortran_order = ndarray.flags.f_contiguous and not ndarray.flags.c_contiguous
```

Comment:
In the tuple instantiation case should we allow the order to be specified ? Or perhaps set a default (rather than leaving it as None) ?